include/swift/Frontend/FrontendOptions.h

```diff
@@ -135,6 +135,9 @@ class FrontendOptions {
 
   /// Include local definitions/references in the index data.
   bool IndexIncludeLocals = false;
+  
+  /// Whether to compress the record and unit files in the index store.
+  bool IndexStoreCompress;
```

Comment:
Already run tests so I wouldn't bother in this PR, but can we default this to `false` 🙏?

Reply:
Yes, here we go: https://github.com/swiftlang/swift/pull/83598